share/functions/__fish_print_rpm_packages.fish

```diff
@@ -31,14 +23,7 @@ function __fish_print_rpm_packages
     if type -q -f rpm
         # If the cache is less than five minutes old, we do not recalculate it
         set -l cache_file $xdg_cache_home/rpm
-        if test -f $cache_file
-            cat $cache_file
-            set -l age (path mtime -R -- $cache_file)
-            set -l max_age 250
-            if test $age -lt $max_age
-                return
-            end
-        end
+        __fish_cache_read $cache_file 250 && return
```

Comment:
there's a subtle behavior change here.
Previously, we used to use the contents of the cached file even when the file is too old.
i.e. the max_age check is only used to decide whether to refresh the cache.
This seems like a sensible semantics for most (all?) callers, no?

Reply:
Good point, I didn't notice that. The only drawback is that when there is no cache file already created, no completions will be returned.